src/pages/index.tsx

```diff
@@ -31,119 +32,107 @@ import {
-    text: `Learn about the architecture of the Uniswap Protocol smart contracts through guided examples`,
-    name: ElementName.SMART_CONTRACT_OVERVIEW,
+    to: '/contracts/v3/overview',
+    text: `Learn about the architecture of the Uniswap Protocol smart contracts through guided examples.`,
   },
 ]
 
-export const github = [
+export const developerLinks = [
   {
     title: 'uniswap-v3-core',
     href: 'https://github.com/Uniswap/uniswap-v3-core',
     icon: CodeIcon,
-    name: ElementName.V3_CORE,
   },
   {
     title: 'uniswap-v3-sdk',
     href: 'https://github.com/Uniswap/uniswap-v3-sdk',
     icon: CodeIcon,
-    name: ElementName.V3_SDK,
   },
   {
     title: 'uniswap-v3-periphery',
     href: 'https://github.com/Uniswap/uniswap-v3-periphery',
     icon: CodeIcon,
-    name: ElementName.V3_PERIPHERY,
   },
   {
     title: 'Deployment addresses',
     href: 'https://github.com/Uniswap/uniswap-v3-periphery/blob/main/deploys.md',
-    name: ElementName.V3_CONTRACT_ADDRESS,
   },
   {
     title: '@uniswap/widgets',
     href: 'https://www.npmjs.com/package/@uniswap/widgets',
-    icon: CodeIcon,
-    name: ElementName.V3_WIDGETS,
   },
 ]
 
-export const guides = [
+export const dAppGuides = [
   {
-    title: 'SDK Quick Start',
-    text: 'Integrate with the Uniswap Protocol using JavaScript',
-    to: './sdk/v3/guides/quick-start',
-    name: ElementName.SDK_QUICK_START,
+    title: 'Setup your environment',
+    text: 'Prepare your local environment by installing the required dependencies',
+    to: '/sdk/v3/guides/quick-start',
   },
   {
-    title: 'Implementing a Swap',
-    text: 'Start swapping from a smart contract in Solidity',
-    to: './contracts/v3/guides/swaps/single-swaps',
-    name: ElementName.IMPLEMENT_SWAP,
+    title: 'Fetch token prices',
+    text: 'Fetch the price of tokens in a specific Pool',
+    to: '/sdk/v3/guides/fetching-prices',
   },
   {
-    title: 'Embedding a Swap Widget',
-    text: 'Let your users trade tokens without leaving your dApp',
-    to: './sdk/swap-widget/overview',
-    name: ElementName.EMBED_SWAP_WIDGET,
+    title: 'Create a Trade',
+    text: 'Fetch a Quote for a Trade and execute the Trade',
+    to: '/sdk/v3/guides/creating-a-trade',
   },
   {
-    title: 'Providing Liquidity',
-    text: 'Provide liquidity from a smart contract in Solidity',
-    to: './contracts/v3/guides/providing-liquidity/setting-up',
-    name: ElementName.PROVIDE_LIQUIDITY,
+    title: 'Route trades',
+    text: 'Use Routing to get optimized prices for your Trades',
+    to: '/sdk/v3/guides/auto-router',
   },
   {
-    title: 'Building an Oracle',
-    text: 'Learn how Uniswap v3 pools can serve as oracles',
-    to: './concepts/protocol/oracle',
-    name: ElementName.BUILD_ORACLE,
+    title: 'Provide liquidity',
+    text: "Contribute to a Pool's liquidity by using tokens to earn fees",
+    to: '/sdk/v3/guides/liquidity/minting',
+  },
+  {
+    title: 'UI Component',
```

Comment:
Can we keep `Embedding a Swap Widget`? What's the benefit of this new title?

Reply:
The benefit is that the user does not know what a swap widget is at this point - covered with Alex on Wednesday